app/views/experiments/_experiment_result.html.haml

```diff
@@ -1,8 +1,8 @@
 %h3.subheader= "Experiment result"
 
 - unless @experiment.results.nil?
```

Comment:
To można zrobić za pomocą JS - być może jest takie coś jak .show() i .hide() na elementach jQuery. Jest też taka klasa CSS .hidden albo .hide - poszukaj w projekcie. Czyli na początku element jest hidden, a jeśli updatujemy i jest jakaś treść to robimy show (najprawdopodobniej usuwamy styl .hidden).